lms/static/scripts/frontend_apps/components/test/BookSelector-test.js

```diff
@@ -274,7 +279,8 @@ describe('BookSelector', () => {
 
     context('book metadata fetch failed', () => {
       it('shows error from service exception', async () => {
-        const error = new Error('Something went wrong');
+        fakeFormatErrorMessage.returns('Something went wrong');
```

Comment:
If you are mocking `formatErrorMessage` here, which I think is optional, it would be good to assert that the correct error is passed to it. Another option is to mock this function to return a string that incorporates `error.message` (eg. `Error: ${error.message}`). That way you can effectively check both things at once.